src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -254,7 +254,7 @@ class ExternalIssueActions extends AsyncComponent {
           onClose={this.deleteIssue.bind(this)}
           integrationType={selectedIntegration.provider.key}
           integrationName={selectedIntegration.name}
-          hoverCardHeader={`Linked ${selectedIntegration.provider.name} Issue`}
+          hoverCardHeader={t(`Linked ${selectedIntegration.provider.name} Issue`)}
```

Comment:
You should use parameterization here, otherwise the translation key will include the provider name

Reply:
ty

Reply:
resolved!